upload/upload.go

```diff
@@ -193,6 +194,24 @@ func NewHandler(
 			B64Identity: b64Identity,
 		}
 
+		var exceedsSizeLimit bool
```

Comment:
Minor nitpick.  You might consider moving this logic into a method to keep the main logic flow more clean / easier to "mentally parse".